README.md

```diff
@@ -61,4 +61,4 @@ We recommend:
 
 ## Publishing
 
-The application will automatically increment the minor build version and publish a release version to the Brightspace CDN after merge to the `master` branch is complete. If you wish to increment the `patch` or `major` version instead please add **[increment patch]** or **[increment major]** to the notes inside your merge message.
+The application uses the [Semantic Release Plugin](https://github.com/BrightspaceUI/actions/tree/master/semantic-release) and is configured to publish to npm.
```

Comment:
We've been adding this huge block of text to all the readmes using semantic release: https://github.com/BrightspaceUILabs/grade-result#versioning--releasing
If you'd rather avoid that, I liked how the `d2l-fetch` readme at least linked directly to the "triggers" part of the `semantic-release` action readme

Reply:
i'm indifferent, i like linking better than copy paste though